compiler-rt/lib/rtsan/rtsan.cpp

```diff
@@ -9,7 +9,8 @@
 //===----------------------------------------------------------------------===//
 
 #include <rtsan/rtsan.h>
-#include <rtsan/rtsan_context.h>
+#include <rtsan/rtsan_assertions.h>
+#include <rtsan/rtsan_diagnostics.h>
```

Comment:
I don't think we need this header, you could also remove the newline change below